docs/user-guide/geoprocessing-tool.md

```diff
@@ -0,0 +1,65 @@
+# GeoProcessing Tool
+
+*******************
+
+This tool allows to perform geographic operations using one or more layers containing geometry and save the result of the operation as a new layer.
+
+!!! note
+    The **GeoProcessing Tool** is not included by default in the MapStore configuration but it can be configured within an [application contexts](application-context.md#configure-plugins) if needed.
+    The plugin works only if the **GeoProcessing WPS service** (available as a community extension) is properly installed in [GeoServer](http://geoserver.org/).
+
+By clicking the **GeoProcessing Tool** <img src="../img/button/geoprocessing-button.jpg" class="ms-docbutton"/> button, available in the [Side Toolbar](mapstore-toolbars.md#side-toolbar), a panel opens so that the user can choose the geographic operations between **Buffer** and **Intersection**.
```

Comment:
```suggestion
By clicking the **GeoProcessing Tool** <img src="../img/button/geoprocessing-button.jpg" class="ms-docbutton"/> button, available in the [Side Toolbar](mapstore-toolbars.md#side-toolbar), a panel opens so that the user can choose the geographic operations between **Buffer** and **Intersection**.
```